uniparc-rest/src/main/java/org/uniprot/api/uniparc/request/UniParcBestGuessRequest.java

```diff
@@ -0,0 +1,37 @@
+package org.uniprot.api.uniparc.request;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+
+import org.uniprot.api.rest.request.QueryFieldMetaReaderImpl;
+import org.uniprot.api.rest.request.ReturnFieldMetaReaderImpl;
+import org.uniprot.api.rest.validation.ValidReturnFields;
+import org.uniprot.api.rest.validation.ValidSolrQueryFields;
+import org.uniprot.api.rest.validation.ValidSolrQuerySyntax;
+import org.uniprot.store.config.UniProtDataType;
+
+import uk.ac.ebi.uniprot.openapi.extension.ModelFieldMeta;
+import io.swagger.v3.oas.annotations.Parameter;
+
+/**
+ * @author lgonzales
+ * @since 12/08/2020
+ */
+@Data
+public class UniParcBestGuessRequest {
+
+    @ModelFieldMeta(reader = QueryFieldMetaReaderImpl.class, path = "uniparc-search-fields.json")
+    @Parameter(description = "Criteria to search the uniparc. It can take any valid solr query.")
```

Comment:
-> The query used to search UniParc. This accepts any valid standard Lucene query.